### PR TITLE
New package: Microsoft.DotNet.AspNetCore.2_1

### PR DESCRIPTION
**Pull request by MisterNoobNoob**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/157224)

### CODE REVIEW

**Comment by MisterNoobNoob** [comment]
@microsoft-github-policy-service agree

File: manifests/m/MicrosoftCorporation/Microsoft/NETCoreRuntime-2/1/30(x64)/2.1.30.30411/MicrosoftCorporation.Microsoft.NETCoreRuntime-2.1.30(x64).locale.de.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.Microsoft.NETCoreRuntime-2.1.30(x64)
```
Comment (stephengillie) [inline manifests/m/MicrosoftCorporation/Microsoft/NETCoreRuntime-2/1/30(x64)/2.1.30.30411/MicrosoftCorporation.Microsoft.NETCoreRuntime-2.1.30(x64).locale.de.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.DotNet.AspNetCore.2_1
```
Please update file name and path to match this change.

File: manifests/m/MicrosoftCorporation/Microsoft/NETCoreRuntime-2/1/30(x64)/2.1.30.30411/MicrosoftCorporation.Microsoft.NETCoreRuntime-2.1.30(x64).installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.Microsoft.NETCoreRuntime-2.1.30(x64)
```
Comment (stephengillie) [inline manifests/m/MicrosoftCorporation/Microsoft/NETCoreRuntime-2/1/30(x64)/2.1.30.30411/MicrosoftCorporation.Microsoft.NETCoreRuntime-2.1.30(x64).installer.yaml:4]:
```suggestion
PackageIdentifier: Microsoft.DotNet.AspNetCore.2_1
```
Please update file name and path to match this change.

File: manifests/m/MicrosoftCorporation/Microsoft/NETCoreRuntime-2/1/30(x64)/2.1.30.30411/MicrosoftCorporation.Microsoft.NETCoreRuntime-2.1.30(x64).yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: MicrosoftCorporation.Microsoft.NETCoreRuntime-2.1.30(x64)
```
Comment (stephengillie) [inline manifests/m/MicrosoftCorporation/Microsoft/NETCoreRuntime-2/1/30(x64)/2.1.30.30411/MicrosoftCorporation.Microsoft.NETCoreRuntime-2.1.30(x64).yaml:4]:
```suggestion
PackageIdentifier: Microsoft.DotNet.AspNetCore.2_1
```
Please update file name and path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @MisterNoobNoob, 

This appears to be another version of a package already in the repo.

**Comment by stephengillie** [comment]
Verify:
image
image
The application shows `2.1.30`, while Apps and features (reading the Registry) shows the full version. When these don't match, the Registry data should be added as a `DisplayVersion`, and the `PackageVersion` updated to match the application. Should this be done here?

**Comment by MisterNoobNoob** [comment]
> Verify: image image The application shows `2.1.30`, while Apps and features (reading the Registry) shows the full version. When these don't match, the Registry data should be added as a `DisplayVersion`, and the `PackageVersion` updated to match the application. Should this be done here?

This will not be necessary.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**